locales/fr.json

```diff
@@ -348,14 +346,16 @@
   "Using CRA software called": "En utilisant le logiciel de l'ARC,",
   "NETFILE": "IMPÔTNET",
   "For more information, contact CRA by:": "Pour de plus amples renseignements, communiquez avec l’ARC par :",
-  "Phone: 1-800-959-8281": "Phone: 1-800-959-8281",
-  "TTY: 1-800-665-0354": "Téléphone : 1-800-665-0354",
+  "Phone: 1-800-959-8281": "Téléphone : 1-800-959-8281",
```

Comment:
You might want to double-check, but from what I saw on the website (https://www.canada.ca/fr/agence-revenu/organisation/coordonnees/numeros-telephone.html), the phone is different for French: 1-800-959-7383